tests/choice_negative/choice_negative.fz

```diff
@@ -31,20 +31,23 @@ choice_negative is
   cyclic1 is
     A : choice A i32 String is  // 1. should flag an error: cyclic choice
 
-  cyclic2 is
-    A : choice (ref A) i32 String is
+  # NYI: this test does no longer make sense: "choice feature must not be ref"
+  # cyclic2 is
+  #   A : choice (ref A) i32 String is
```

Comment:
But even then they still cause errors, which according to the comments in the test they should not.

```
--CURDIR--/choice_negative.fz:36:5: error 16: choice feature must not be ref
    A ref : choice A i32 String is
----^
A choice feature must be a value type since it is not constructed 

…

--CURDIR--/choice_negative.fz:43:5: error 18: choice feature must not be ref
    A ref : choice i32 A String is
----^
A choice feature must be a value type since it is not constructed 

…

--CURDIR--/choice_negative.fz:50:5: error 20: choice feature must not be ref
    A ref : choice i32 String A is
----^
A choice feature must be a value type since it is not constructed 
```

Reply:
@simonvonhackewitz I think then the tests needs to be changed to reflect that these now cause an error.